packages/block-library/src/post-excerpt/index.php

```diff
@@ -24,7 +24,16 @@ function render_block_core_post_excerpt( $attributes, $content, $block ) {
 		return '';
 	}
 
-	$more_text           = ! empty( $attributes['moreText'] ) ? '<a class="wp-block-post-excerpt__more-link" href="' . esc_url( get_the_permalink( $block->context['postId'] ) ) . '">' . wp_kses_post( $attributes['moreText'] ) . '</a>' : '';
+	$post_ID            = $block->context['postId'];
+	$post_title         = get_the_title( $post_ID );
+	$allowed_html       = array( 'abbr' => array(), 'acronym' => array(), 'b' => array(), 'br' => array(), 'cite' => array(), 'code' => array(), 'em' => array(), 'i' => array(), 'span' => array(), 'strong' => array(), 'sub' => array(), 'sup' => array(), 'time' => array() );
```

Comment:
@joedolson I included only a subset of inline elements because some didn't make sense e.g. `input`, `a`.

Also it feels like this maybe should be abstracted, is there a place where a utility function like this should live? Since it will be re-used for the Read More block also.

Reply:
Maybe we should look at adding a new parameter to `wp_kses_allowed_html` that specifically permits inline elements. `wp_kses_allowed_html( 'data' )` is close to what we need, but it's not an exact match.